avidemux_plugins/ADM_videoFilters6/crop/qt5/DIA_flyCrop.cpp

```diff
@@ -625,7 +646,7 @@ Ui_cropWindow::Ui_cropWindow(QWidget* parent, crop *param,ADM_coreVideoFilter *i
     connect(pushButtonReset,SIGNAL(clicked(bool)),this,SLOT(reset(bool)));
 
     const QString autorun = QT_TRANSLATE_NOOP("crop","Auto Crop");
-    pushButtonAutoCrop = ui.buttonBox->addButton(autorun,QDialogButtonBox::ActionRole);
+    pushButtonAutoCrop = ui.buttonBox->addButton(autorun,QDialogButtonBox::ResetRole);    // group together with the Reset button
```

Comment:
It would be great not to compromise code semantics to implement minor usability or purely cosmetical tweaks, especially when the perceived issue results from Qt following system guidelines (e.g. QDialogButtonBox::ActionRole is already placed on the left side on macOS). IMHO, it means actually introducing bugs to counteract something which is sometimes unfortunate.

Please revert this one change.